src/app/routes/analytics.js

```diff
@@ -7,4 +8,14 @@ module.exports = (Router) => {
     const { actionName } = req.body;
     AnalyticsService.actions[actionName](req);
   });
+
+  Router.post('/data/p', page, (req, res) => {
+    res.status(200).send();
+  });
+
+  Router.post('/data/t', (req, res) => {
+    res.status(200).send();
+    const { actionName } = req.body;
+    AnalyticsService.actions[actionName](req);
```

Comment:
If analytics volume is going to be big, add a `try-catch` here to avoid a logs overflow of the catch-all handler if a lot of errors happen.